tools/lxdclient/client.go

```diff
@@ -52,11 +56,7 @@ var lxdLoadConfig = lxd.LoadConfig
 func newRawClient(remote, configDir string) (*lxd.Client, error) {
 	logger.Debugf("loading LXD client config from %q", configDir)
 
-	// This will go away once LoadConfig takes a dirname argument.
-	origDirname := updateLXDVars(configDir)
-	defer updateLXDVars(origDirname)
-
-	cfg, err := lxdLoadConfig()
+	cfg, err := lxdLoadConfig(path.Join(configDir, "config.yml"))
```

Comment:
So we do have the issue that we don't generally let Providers (and thus Broker/container.Managers) have direct DB access. So we'd need to be passing this information in via just extra parameters to Init or Config.
What are the things we need, certificates and the config that says to use "local"?
I know we have support for some of those things at the Provider level, to date we haven't needed them for Brokers. Will need some thought as you need a custom cert for every machine to every LXD, right? How do we validate the LXD's certificate, or do we just accept it as long as it is valid?